src/workflow/examples/activity-sequence.ts

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import WorkflowClient from "../client/WorkflowClient";
+import WorkflowActivityContext from "../runtime/WorkflowActivityContext";
+import WorkflowContext from "../runtime/WorkflowContext";
+import WorkflowRuntime from "../runtime/WorkflowRuntime";
+import { TWorkflow } from "../types/Workflow.type";
+
+(async () => {
```

Comment:
Why do you need to do this?

Reply:
This is a working way that I found to run the example using command like `npx ts-node ./examples/activity-sequence.ts`, let me know if there is a better way to run the example!

Reply:
Users can run the examples by following steps at https://github.com/dapr/js-sdk/blob/d1619a274dba6ad69773ed7f7c9f25336cde30c0/examples/workflow/authoring/README.md